### DISCUSSION TITLE
Importing pycuda.driver under sudo leads to "libcurand.so.10: cannot open shared object file: No such file or directory"

### DISCUSSION THREAD

**Discussion by sliang11** [inducer/pycuda#353] [Troubleshooting]
I am trying to import pycuda-2021.1 in a python script. My OS is Ubuntu 18.04. I have cuda toolkit 11.2 installed and my nvidia driver version is 460.27.04. My python interpretor is Python 3.8. When I execute

`import pycuda.driver as cuda`

I seem to be able to do so without executing the python script under sudo. However, when using sudo, this leads to the following error:

> ImportError: libcurand.so.10: cannot open shared object file: No such file or directory

How can I resolve this issue?

**Comment by inducer** [comment]
How did you install pycuda?

**Comment by inducer** [comment]
Generally, if you're facing installation issues, my recommendation would be to install pycuda from conda-forge:

https://github.com/conda-forge/pycuda-feedstock
